facturacion_mexico/facturacion_fiscal/tests/test_writer_timbrado_uuid.py

```
@@ -0,0 +1,255 @@
+
+		# Ejecutar la FASE 3 real con sus boundaries de IO neutralizados.
+		from facturacion_mexico.facturacion_fiscal.timbrado_api import TimbradoAPI
+
+		si_doc = frappe.get_doc("Sales Invoice", si)
+		ffm_doc = frappe.get_doc("Factura Fiscal Mexico", ffm)
+
+		# Neutralizar SOLO la escritura sobre la Sales Invoice mínima (validación mandatory);
+		# la escritura sobre el FFM (lo que se prueba) se deja real.
+		_orig_set_value = frappe.set_value
+
+		def _set_value_si_noop(dt, dn, *args, **kwargs):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_set_value_si_noop`

(ANN202)